lib/elixir_newbie_web/live/resources_live.ex

```diff
@@ -3,11 +3,16 @@ defmodule ElixirNewbieWeb.ResourcesLive do
 
   def render(assigns) do
     ~H"""
-    Resources
+    <section class="text-white">
+      <p>Resources</p>
+      <%= for post <- @posts do %>
+        <%= raw post.body %>
+      <% end %>
+    </section>
     """
   end
 
   def mount(_params, _session, socket) do
-    {:ok, socket}
+    {:ok, assign(socket, [posts: ElixirNewbie.Blog.all_posts])}
```

Comment:
I didn't show you this syntax, but this is a good time to mention it.

You can define a single value using `assign/3` instead of `assign/2`.

```elixir
posts = ElixirNewbie.Blog.all_posts()
assign(socket, :posts, posts)
```

You can also pipe these calls together.

```elixir
socket
|> assign(:key1, "value1")
|> assign(:key2, "value2")
```